skops/card/tests/test_card.py

```diff
@@ -169,6 +173,70 @@ def test_add_plot(destination_path, model_card):
     assert "![fig1](fig1.png)" in model_card
 
 
+def test_permutation_importances(
+    iris_estimator, iris_data, model_card, destination_path
+):
+    X, y = iris_data
+    result = permutation_importance(
+        iris_estimator, X, y, n_repeats=10, random_state=42, n_jobs=2
+    )
+
+    model_card.add_permutation_importances(
+        result,
+        X.columns,
+        Path(destination_path) / "importance.png",
+        "Permutation Importance",
+    )
+    temp_path = Path(destination_path) / "importance.png"
+    assert f"![Permutation Importance]({temp_path}" in model_card.render()
+
+
+def test_matplotlib_dependency_error(
+    iris_estimator, iris_data, model_card, destination_path
+):
+    with mock.patch.dict(sys.modules):
+        sys.modules["matplotlib"] = None
+        X, y = iris_data
+        result = permutation_importance(
+            iris_estimator, X, y, n_repeats=10, random_state=42, n_jobs=2
+        )
+        with pytest.raises(ModuleNotFoundError):
```

Comment:
Ideally, also check the error message, to ensure that the specific error is indeed raised.